src/test/ui/command-argv0-debug.rs

```diff
@@ -0,0 +1,24 @@
+// run-pass
```

Comment:
Please move this test to some sub-folder. Dumping things right in `ui/` clutters the overall organization of tests.

Reply:
OK - it's adjacent to the existing tests for command. Should I move them all?

Reply:
That would be great, thanks!

Reply:
OK, done.